account_statement_import_online/models/online_bank_statement_provider.py

```diff
@@ -95,6 +95,10 @@ class OnlineBankStatementProvider(models.Model):
     certificate_public_key = fields.Text()
     certificate_private_key = fields.Text()
     certificate_chain = fields.Text()
+    dont_create_empty_statements = fields.Boolean(
+        string="Don't create empty statements",
+        default=True,
```

Comment:
Default should be false, OR it should be `allow_empty_statements` - to avoid sneaky change.

Reply:
No, we want this default. You can change it in your side.

Reply:
What about proper mentioning in readme? So that users would know about the sneaky change?

Reply:
Also, wouldn't it be more readable to do `allow_empty_statements` with  default false then?

Reply:
I'd really expect a major version version bump with this also, since breaking change, in addition to explaining in README.

Reply:
> No, we want this default. You can change it in your side.

Ok